src/realm/query_expression.hpp

```diff
@@ -2598,6 +2768,11 @@ class Columns : public Subexpr2<T> {
         return m_sg ? get_column_base().get_column_index() : m_column_ndx;
     }
 
+    SizeOperator<Size<T>> size()
+    {
+        return SizeOperator<Size<T>>(this->clone(nullptr));
```

Comment:
Given that you have to statically know the type of the column you're operating on, it's hard to see how anyone will ever sensibly write a query asking for the size of a `Columns<Int>`.